web/frontend/src/app.vue

```diff
@@ -11,22 +10,6 @@ const {
   webrtcSignalingAddress,
 } = window;
 
-const rtcConfig = {
```

Comment:
where's the RTC config now for the local case?

Reply:
@maximpertsov is this just in the TS SDK itself?

Reply:
No, the only thing we changed recently (as of version `0.0.46`) was automatically fetching additional config.

Reply:
okay so se may need to bring this back for local case.

Reply:
ah great